kde-apps/kpimtextedit/kpimtextedit-9999.ebuild

```diff
@@ -20,9 +20,9 @@ IUSE="speech"
 RESTRICT="test"
 
 RDEPEND="
-	>=dev-qt/qtbase-${QTMIN}:6[gui,widgets]
+	dev-libs/ktextaddons:6[speech?]
```

Comment:
Since you're touching this, please test emerging kpimtextedit[-speech] with `dev-libs/ktextaddons:6[speech]::installed`.

Reply:
Works as expected, with $(cmake_use_find_package speech KF6TextEditTextToSpeech) making sure it only gets enabled if the use flag is on.

But reverse dependencies with speech= may need a look as the cmake_use_find_package approach should work for them most likely as well.

Reply:
> Works as expected, with $(cmake_use_find_package speech KF6TextEditTextToSpeech) making sure it only gets enabled if the use flag is on (no automagic!).

I know how it works, the problem would be ktextaddons::installed cmake module find_dependency calls affected by this.